### ISSUE TITLE
seems not to work

### ISSUE THREAD

**Issue by anybany** [MissVeronica/um-redirect-login-log#2]
Hi, i am trying to use your plugin to debug UM.
i can't find the settings under misc you mention here.
i am using the shortcode [um_events_trace_log] on a single page.
could you please check if your plugin is compatible with the newest UM version?

**Comment by MissVeronica** [comment]
Yes you are right about the version 3.3 settings and I have a new version 3.4 available where settings been moved to a new Status plugin for the listing of the events logged and the shortcode is not used any more.
I will make these plugins available for you to download.

**Comment by MissVeronica** [comment]
I have found the issue with the version 3.3 and the settings in `Misc`
Rename the plugin directory to only use lowercase characters ie `um-events-trace-log-main`
I will work with the Status plugin and my intention is to merge these two plugins.

**Comment by anybany** [comment]
yes, renaming the folder worked. 
and your plugin already helped me to find a bad redirect :)
Thank you!
